go_pprof_scraper/README.md

```diff
@@ -0,0 +1,67 @@
+# Agent Check: Go-pprof-scraper
+
+## Overview
+
+This check collects profiles from Go applications which expose the [`/debug/pprof`][1] endpoint.
+
+**NOTE**: Prefer instrumenting services with the [dd-trace-go][12] profiling client library. The client library
+offers better integration with other Datadog services, such as through [code hotspots and enpoint filtering][13].
+This integration should be used for services for which you do not control the source code.
+
+**NOTE**: Using this integration will result in billing for hosts under Datadog's [Continuous Profiler][10] service.
+For more on Continuous Profiler billing, consult our [billing documentation][11].
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+TODO: update this
+The Go-pprof-scraper check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `go_pprof_scraper.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your go_pprof_scraper performance data. See the [sample go_pprof_scraper.d/conf.yaml][4] for all available configuration options.
+
+2. [Restart the Agent][5].
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `go_pprof_scraper` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][7] for a list of metrics provided by this integration.
```

Comment:
Looks like your csv file is empty. In this case you can say that this integration does not collect metrics

Reply:
Updated